src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbradiobutton.directive.js

```diff
@@ -0,0 +1,55 @@
+/**
+@ngdoc directive
+@name umbraco.directives.directive:umbRadiobutton
+@restrict E
+@scope
+
+@description
+<b>Added in Umbraco version 7.14.0</b> Use this directive to render an umbraco radio button.
+
+<h3>Markup example</h3>
+<pre>
+    <div ng-controller="My.Controller as vm">
+
+        <umb-radiobutton
+            name="checkboxlist"
```

Comment:
@nielslyngsoe the `name` in the example here should probably be changed from `checkboxlist` to `radiobuttonlist`.